examples/dashboards/bucket_replicate.json

```diff
@@ -47,7 +47,7 @@
                "steppedLine": false,
                "targets": [
                   {
-                     "expr": "sum(rate(thanos_replicate_replication_runs_total{result=\"error\", namespace=\"$namespace\",job=~\"thanos-bucket-replicate.*\"}[$interval])) / sum(rate(thanos_replicate_replication_runs_total{namespace=\"$namespace\",job=~\"thanos-bucket-replicate.*\"}[$interval]))",
+                     "expr": "sum(rate(thanos_replicate_replication_runs_total{result=\"error\", job=\"$job\"}[$interval])) / sum(rate(thanos_replicate_replication_runs_total{job=\"$job\"}[$interval]))",
```

Comment:
should we do `by (job, namespace)` here?

Reply:
Let me comment jsonnet code instead, but this example can mislead people - some of them maybe even copies those example 🤔